openbb_terminal/cryptocurrency/defi/defipulse_model.py

```diff
@@ -12,6 +12,23 @@
 logger = logging.getLogger(__name__)
 
 
+@log_start_end(log=logger)
+def num_to_int(num):
+    if num[-1] == "M":
+        return int(float(num[1:-1])) * 1000000
+    if num[-1] == "B":
+        return int(float(num[1:-1])) * 1000000000
+    if num[-1] == "k":
+        return int(float(num[1:-1])) * 1000
```

Comment:
small change.  Probably want to make sure that we look at .upper() and match the "k" to "K".

Reply:
I thought thousand was normally lowercase?